apps/financial-aid/web-veita/src/utils/useApplicationState.ts

```diff
@@ -1,28 +1,38 @@
 import { useContext } from 'react'
 import { useMutation } from '@apollo/client'
 import {
+  Application,
   ApplicationEventType,
+  ApplicationFilters,
   ApplicationState,
-  UpdateApplicationResponseType,
 } from '@island.is/financial-aid/shared/lib'
-import { UpdateApplicationMutation } from '../../graphql'
 import { ApplicationFiltersContext } from '../components/ApplicationFiltersProvider/ApplicationFiltersProvider'
 import { AdminContext } from '../components/AdminProvider/AdminProvider'
-
-interface SaveData {
-  updateApplicationRes: UpdateApplicationResponseType
-}
+import {
+  UpdateApplicationMutation,
+  ApplicationFiltersMutation,
+} from '../../graphql/sharedGql'
```

Comment:
@island.is/financial-aid/ nitty pick, however in next pull request will this be under utils/hooks, and then it is nice not having to change the imports 👼